src/main/java/bio/terra/testrunner/common/utils/KubernetesClientUtils.java

```diff
@@ -285,6 +297,50 @@ public static void buildKubernetesClientObjectWithClientKey(
 
     kubernetesClientCoreObject = new CoreV1Api();
     kubernetesClientAppsObject = new AppsV1Api();
+
+    // Import MCTerra Component versions from ConfigMap and outputs JSON-ready format
+    importComponentVersions();
```

Comment:
I think this should be called from the TestRunner class instead of in the setup method here. (e.g. after setup and before modifying the cluster, around line 164).

Reply:
Yup I think TestRunner can make an explicit call to `getComponentVersions()` so the `importComponentVersions()` method get called inside `getComponentVersions()`.

The code has been reorganized to take this into consideration.